crypto/src/test/java/com/virgilsecurity/sdk/crypto/VirgilCryptoTest.java

```diff
@@ -372,16 +386,19 @@ public void importPublicKey(VirgilCrypto crypto) throws CryptoException {
     }
   }
 
-  @SignCryptoTest
+  @ParameterizedTest
+  @MethodSource("comparableKeyTypesCryptos")
   public void sign_stream_compareToByteArraySign(VirgilCrypto crypto) throws CryptoException {
     VirgilKeyPair keyPair = crypto.generateKeyPair();
     byte[] signature = crypto.generateSignature(TEXT.getBytes(), keyPair.getPrivateKey());
+//    byte[] signature2 = crypto.generateSignature(TEXT.getBytes(), keyPair.getPrivateKey());
```

Comment:
Why do we need this check?

Reply:
We had a bug when signature of stream and signature of byte[] were different